books/RayTracingTheRestOfYourLife.html

```diff
@@ -1102,31 +1128,42 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
         }
         std::cout << std::fixed << std::setprecision(12);
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
         std::cout << "I = " << sum/N << '\n';
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
     }
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    [Listing [integ-xsq-2]: <kbd>[integrate_x_sq.cc]</kbd> Integrating $x^2$ with PDF]
-</div>
+    [Listing [integ-xsq-2]: <kbd>[integrate_x_sq.cc]</kbd> Explicit uniform PDF for $x^2$]
 
 
-Importance Sampling
---------------------
-Since we are sampling more where the PDF is big, we might expect less noise and thus faster
-convergence. In effect, we are steering our samples toward the parts of the distribution that are
-more _important_. This is why using a carefully chosen nonuniform PDF is usually called _importance
-sampling_.
+There are a couple of important things to emphasize. Every value of $x$ represents one sample of the
+function $x^2$ within the distribution [0,2]. We use a function $f$ to randomly select samples from
+within this distribution. We are no longer multiplying the output sum by $2$ to arrive at the final answer. We were previously multiplying the average over the interval (sum/N) times the length of the interval ($b - a = 2$). Here, we don't need to multiply by the interval length.
+
+
+We need to account for the nonuniformity of the PDF of $x$. Failing to account for this
+nonuniformity will introduce bias in our scene, indeed this bias is the source of our inaccurately
+bright image, so if we account for nonuniformity, we will get accurate results. The PDF will "steer"
+samples toward specific parts of the distribution, which will cause us to converge faster, but this
+will introduce bias. To remove bias we need to down-weight where we sample more frequently, and
+to up-weight where we sample less frequently. For our new nonuniform random number generator the PDF
+defines how much or how little we sample a specific portion. So the weighting function should be
+proportional to $1/\text{pdf}$. In fact it is exactly $1/\text{pdf}$. This is why we divide `x*x` by
+`pdf(x)`.
+</div>
 
 <div class='together'>
-We can choose whatever PDF we want. If we take that same code from before and set a uniform PDF of
-PDF = $1/2$ over the range [0,2], then the CDF is $P(x) = x/2$, so $f(d) = 2d$:
+We can try to solve for the integral using the linear PDF $p(r) = \frac{r}{2}$ that we were able to
+solve for both the CDF for, and the inverse of the CDF. If we do that, all we need to do is rewrite
```

Comment:
“… linear PDF $p(r) = \frac{r}{2}$, for which we were able to solve both the CDF and its inverse.”